organization/tests/test_functional.py

```diff
@@ -0,0 +1,48 @@
+from django.core.urlresolvers import reverse
+from django.test import Client, TestCase
+
+from organization.models import Organization
+
+
+class OrganizationTest(TestCase):
+    fixtures = ['user.json']
+
+    def setUp(self):
+        self.client = Client()
+        self.client.login(username='user', password='user')
+
+    def test_create_event(self):
+        url = reverse('create_organization')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+        data = {'name': 'Speakerfight Corp', 'about': 'Cool company'}
+        url = reverse('create_organization')
+        response = self.client.post(url, data)
+        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(
+            response,
+            reverse('update_organization', kwargs={'slug': 'speakerfight-corp'})
+        )
+        self.assertEqual(Organization.objects.count(), 1)
```

Comment:
Can you please assert the organization name and description?

Reply:
For sure \o

Reply:
Done